compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -256,7 +256,7 @@ bool is_NCHW_with_const(const luci::CircleMul *node, luci::CircleNode *&pred_nod
   }
 
   const auto const_rank = multiplier->rank();
-  if (const_rank != 4)
+  if (const_rank != 4 || pred_node->rank() != 4)
```

Comment:
This is OK but it seems better to check `pred_node->rank() != 4` after above `else` block when `pred_node` is set.